unittests/tools/test_auditjs_parser.py

```diff
@@ -65,9 +65,9 @@ def test_auditjs_parser_empty_with_error(self):
             parser = AuditJSParser()
             parser.get_findings(testfile, Test())
             testfile.close()
-            self.assertTrue(
-                "Invalid JSON format. Are you sure you used --json option ?" in str(context.exception)
-            )
+        self.assertTrue(
+            "Invalid JSON format. Are you sure you used --json option ?" in str(context.exception)
+        )
```

Comment:
Looks like this could be an `assertIn`, or no?

```suggestion
        self.assertIn(
            "Invalid JSON format. Are you sure you used --json option ?", str(context.exception)
        )
```